src/coreclr/tools/aot/DependencyGraphViewer/DependencyGraphViewer.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppToolCurrent)-windows</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>DependencyLogViewer</RootNamespace>
+
+    <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
app.manifest should not be required with .NET Core.

Reply:
The tool needs to run elevated - without elevation it will just fail.

Reply:
Ok, I have missed the elevation part.